src/reducers/requestRepoReducer.ts

```diff
@@ -0,0 +1,66 @@
+import { Action } from 'redux-actions';
+
+import { prepareSystemIntakeForApp } from 'data/systemIntake';
+import { fetchRequestRepoIntakes } from 'types/routines';
+import { SystemIntakeForm } from 'types/systemIntake';
+import { isIntakeClosed, isIntakeOpen } from 'utils/systemIntake';
+
+type RequestRepoState = {
+  openIntakes: SystemIntakeForm[];
+  closedIntakes: SystemIntakeForm[];
+  isLoading: boolean | null;
+  error: string | null;
+};
+
+const initialState: RequestRepoState = {
+  openIntakes: [],
+  closedIntakes: [],
+  isLoading: null,
+  error: null
+};
+
+function requestRepoReducer(
+  state = initialState,
+  action: Action<any>
+): RequestRepoState {
+  switch (action.type) {
+    case fetchRequestRepoIntakes.REQUEST:
+      return {
+        ...state,
+        isLoading: true
+      };
+    case fetchRequestRepoIntakes.SUCCESS: {
+      const openIntakes: SystemIntakeForm[] = [];
+      const closedIntakes: SystemIntakeForm[] = [];
+      action.payload.forEach((intake: any) => {
+        if (isIntakeOpen(intake.status)) {
+          openIntakes.push(prepareSystemIntakeForApp(intake));
+        } else if (isIntakeClosed(intake.status)) {
```

Comment:
We haven't played the stories yet for the closed tab of the repository, so we're getting ahead of ourselves a bit here. I'm ok merging this, but we might need to rework it some more.